cypress/integration/InHidingPage.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Are you or your relatives in hiding or in danger page
+  As a user
+  I want to see options on my hiding or danger status
+  So I can check my eligibility
+
+  Background:
+    Given I visit the "relatives in hiding" page
+
+  Scenario: Progression Bar shows check eligibility
+    Then the progression bar is at "check eligibility"
+
+  Scenario: Has a back link to navigate to the where to start looking page
+    When I click the "back" link
+    Then I am on the "where should we start looking" page
+
+  Scenario: Selecting No Im not worried about searching in public and clicking continue navigates to the Eligible for Both Services page
+    When I select "no Im not worried about searching in public" radio button
+    Then I click the "continue" button
+    Then I am on the "eligible for both" page
```

Comment:
I think quotes around the bit their selecting helps differentiate it as a thing from the rest of the words, also, I think it looks weird without the apostrophe, is there a reason for that?

```suggestion
  Scenario: Selecting "No I'm not worried" about searching in public and clicking continue navigates to the Eligible for Both Services page
    When I select "no i'm not worried about searching in public" radio button
    Then I click the "continue" button
    Then I am on the "eligible for both" page
```

Same applies in rest of this file. Happy to jump on a call and thrash this out if that's quickest.